charts/mainflux/templates/adapter_coap-deployment.yaml

```
@@ -20,7 +20,7 @@ spec:
         - env:
             - name: MF_JAEGER_URL
               value: {{ .Release.Name }}-jaeger-operator-jaeger-agent:{{ .Values.defaults.jaegerPort }}
-            - name: MF_NATS_URL
+            - name: MF_BROKER_URL
               value: nats://{{ .Release.Name }}-nats-client:{{ .Values.defaults.natsPort }}
```

@chen-dq , 
The nats broker in the values.yaml file is referred to as `nats`, which means that the hostname of the nats broker will be {{ .Release.Name}}-nats. Can you please update this change in all the relevant locations?

```suggestion
            - name: MF_BROKER_URL
              value: nats://{{ .Release.Name }}-nats:{{ .Values.defaults.natsPort }}   
```